server/shared-kernel/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -95,7 +95,7 @@ IConfigurationSection wordPressConfigurationSection
         {
             // defaults (for dev environment) - this is not the same as Production!!
             string connectionString = string.IsNullOrWhiteSpace(permissionDbConnectionString)
-                ? "server=wordpress-db;uid=root;pwd=octavadev;database=octavapermissons"
```

Comment:
Permissions were moved into the initiatives database?

Reply:
yup.  I was thinking no point in a 1 table database and it is directly related to initiatives, so thought that would be a good place.  Still gets a different Entity Framework context though.